client/utils/validateToken.js

```diff
@@ -0,0 +1,66 @@
+import qs from "qs";
```

Comment:
pay attention, the name of the file is not consistent with the rest of the codebase